exercises/camicia/instructions.md

```diff
@@ -0,0 +1,83 @@
+# Instructions
+
+In this exercise, you will simulate a game very similar to the classic card game **Camicia**.
+Your program will receive the initial configuration of two players' decks and must simulate the game until it ends (or detect that it will never end).
+
+## Rules
+
+- The deck is split between **two players**.
+  The player's cards are read from left to right, where the leftmost card is the top of the deck.
+- A round consists of both players playing at least one card/turn.
```

Comment:
```suggestion
- A round consists of both players playing at least one card.
```

If the round begins with `"K", "5"`, player A plays K, player B pays 5 and player B never gets a "turn".